RxSwift/Subjects/ReplaySubject.swift

```diff
@@ -59,7 +59,7 @@ public class ReplaySubject<Element>
     }
     
     /// Returns observer interface for subject.
-    public func asObserver() -> SubjectObserverType {
+    public func asObserver() -> ReplaySubject<Element> {
```

Comment:
As per the other PR, this was safe because ReplaySubject is `public` (but not `open`) and the only subclasses are internal to RxSwift and aren't public, such as ReplayBufferBase, ReplayOne, etc